view/theme/vier/mobile.css

```diff
@@ -112,14 +112,14 @@ nav ul {
   }
   iframe.embed_rich { width: 100% !important }
     /* TODO: check if this is needed.
-   * oembedded iframe usually have "width" and "height" as attributes
+   * embedded iframe usually have "width" and "height" as attributes
```

Comment:
again, there were some prefixes that seemed intentional (`f`/`e` iirc?), but this `o` didn't seem like one of them

Reply:
"oembed" is the name of the technology, please double check this everywhere.
- https://en.wikipedia.org/wiki/OEmbed